v1/src/main/java/com/google/cloud/teleport/spanner/DdlToAvroSchemaConverter.java

```diff
@@ -240,6 +247,120 @@ public Collection<Schema> convert(Ddl ddl) {
       schemas.add(schema);
     }
 
+    for (PropertyGraph propertyGraph : ddl.propertyGraphs()) {
+      LOG.info("DdlToAvro PropertyGraph {}", propertyGraph.name());
+      SchemaBuilder.RecordBuilder<Schema> recordBuilder =
+          SchemaBuilder.record(generateAvroSchemaName(propertyGraph.name()))
+              .namespace(this.namespace);
+
+      recordBuilder.prop(SPANNER_NAME, propertyGraph.name());
+      recordBuilder.prop(GOOGLE_FORMAT_VERSION, version);
+      recordBuilder.prop(GOOGLE_STORAGE, "CloudSpanner");
+      recordBuilder.prop(SPANNER_ENTITY, SPANNER_ENTITY_PROPERTY_GRAPH);
+
+      // Encode nodeTables
```

Comment:
Can be broken down into 4 separate methods. I know the other lines in this method is also written this way, but propertyGraph is one feature and handling for one feature itself is getting big here. Difficult to follow and debug.

It will be easier to test also.

Reply:
Done.